### DISCUSSION TITLE
ClassNotFoundError docker install

### DISCUSSION THREAD

**Discussion by compgeniuses** [SolidInvoice/SolidInvoice#1163] [Q&A] [answered]
I have built a solidinvoice app for Nethserver 8 here https://github.com/geniusdynamics/ns8-solidinvoice based on the provided docker image.

However, accessing the web ui, prompts me with an error message 
![image](https://github.com/user-attachments/assets/f0a708ec-c48a-42af-a566-19f44e760193)
Wgat could be the provlem

**Comment by pierredup** [comment, marked as answer]
It's most likely that you are running SolidInvoice in dev mode, but the development dependencies are not installed.

You can set the following environment variables to ensure it's running in prod mode:

```
SOLIDINVOICE_ENV=prod
SOLIDINVOICE_DEBUG=0
```

**Reply by compgeniuses** [reply]
HEllo Thank you for your prompt response.

**Comment by compgeniuses** [comment]
IS there not an automated way to have the app pre-configured with the relevant ENV for DB here 
![image](https://github.com/user-attachments/assets/73a0d828-873d-47d3-a701-f53eb3d60275)

**Reply by pierredup** [reply]
Since versio 2.3+, this should be much easier to auto-configure the DB using an env variable (The app already uses the `SOLIDINVOICE_DATABASE_URL` env var, so we can just auto-configure the DB when this is set). I've created #1637 to keep track of this request
